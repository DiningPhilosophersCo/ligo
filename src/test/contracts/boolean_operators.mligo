// Test CameLIGO boolean operators

let or_true (b : bool) : bool =
   b or True

let or_false (b : bool) : bool =
   b or False

let and_true (b : bool) : bool =
   b and True

let and_false (b : bool) : bool =
   b and False

let not_bool (b: bool) : bool =
   not b
