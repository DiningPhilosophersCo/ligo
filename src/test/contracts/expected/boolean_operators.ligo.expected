function or_true (const b : bool) : bool is b or true

function or_false (const b : bool) : bool is b or false

function and_true (const b : bool) : bool is b and true

function and_false (const b : bool) : bool is b and false

function not_bool (const b : bool) : bool is not b
